From jeffm@suse.com  Thu Jan 14 17:17:33 2010
From: Jeff Mahoney <jeffm@suse.com>
Date: Mon, 11 Jan 2010 10:54:31 -0500
Subject: Staging: vme/tsi148: Depend on VIRT_TO_BUS
To: gregkh@suse.com
Message-ID: <20100111160005.264760781@suse.com>


This driver depends on virt_to_bus working correctly, but it doesn't
exist on ppc64 (and probably other arches).

Signed-off-by: Jeff Mahoney <jeffm@suse.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/vme/bridges/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/vme/bridges/Kconfig
+++ b/drivers/staging/vme/bridges/Kconfig
@@ -8,6 +8,7 @@ config VME_CA91CX42
 
 config VME_TSI148
 	tristate "Tempe"
+	depends on VIRT_TO_BUS
 	help
 	 If you say Y here you get support for the Tundra TSI148 VME bridge
 	 chip.
